clone_from/action.yml

```diff
@@ -40,22 +42,23 @@ runs:
       cat ~/.gitconfig
 
       # Checkout if needed
-      if [[ '${{ inputs.checkout }}' != 'true' ]]; then
-        GIT_CLONE_ARGS="--no-checkout"
+      export GIT_ARGS=''
```

Comment:
Minor nit: the quote types should match the rest of the file. It looks like in most cases we're doing `"`. Other than that this looks good to me.